src/gui/Layout.cpp

```diff
@@ -178,176 +179,150 @@ double Layout::getLayoutWidth()
  */
 const int XOURNAL_PADDING = 10;
 
+/**
+ * Allowance for shadow between page pairs in dual mode
+ */
+const int XOURNAL_ROOM_FOR_SHADOW = 3;
+
 /**
  * Padding between the pages
  */
 const int XOURNAL_PADDING_BETWEEN = 15;
 
+	
+
 void Layout::layoutPages()
 {
 	XOJ_CHECK_TYPE(Layout);
 
-	int y = 0;
-
 	int len = this->view->viewPagesLen;
 
 	Settings* settings = this->view->getControl()->getSettings();
-	bool verticalSpace = settings->getAddVerticalSpace();
-	bool horizontalSpace = settings->getAddHorizontalSpace();
+	bool verticalSpace = settings->getAddVerticalSpace();				//TODO: Use these again
+	bool horizontalSpace = settings->getAddHorizontalSpace();		//TODO: Use these again
 	bool dualPage = settings->isShowTwoPages();
```

Comment:
Once more `dualPage` is the flag for all column views? Rename it to `isColumnView` or something like that.